src/tools/iterations.ts

```diff
@@ -54,6 +55,21 @@ export class IterationTools extends BaseTools {
 			async (params) => await tools.searchIterations(params),
 		);
 
+		server.tool(
+			"create-iteration",
+			"Create a new Shortcut iteration",
+			{
+				name: z.string().describe("The name of the iteration"),
+				description: z.string().optional().describe("The description of the iteration"),
+				startDate: z.string().describe("The start date of the iteration"),
+				endDate: z.string().describe("The end date of the iteration"),
+				groupId: z.string().describe("The ID of the group to assign the iteration to"),
+			},
+			async ({ name, description, startDate, endDate, groupId }) => {
```

Comment:
And for clarity - `groupId` here is less important to rename to `teamId` because this is internal to the code. But the `zod` validation above is exposed to the AI as the public API, so it should have consistent usage of terms.